app/models/concerns/file_set/geo_file_format_behavior.rb

```diff
@@ -38,20 +30,23 @@ def vector_work?
   end
 
   module ClassMethods
-    def image_file_formats
-      ['TIFF', 'IMAGE_FILE']
-    end
-
-    def raster_file_formats
-      ['TIFF_GeoTIFF', 'RASTER_FILE']
+    def gdal_formats
+      [
+        'image/tiff; gdal-format=GTiff',
+        'text/plain; gdal-format=AIGrid'
+      ].freeze
     end
 
-    def vector_file_formats
-      ['SHAPEFILE', 'SHAPEFILE_ZIP', 'VECTOR_FILE']
+    def ogr_formats
+      [
+        'application/zip; ogr-format=ESRI_Shapefile',
```

Comment:
Rather than making this more complicated, how about we only put double quotes around formats that are multiple words with a space. From what I can tell, "ESRI Shapefile" is the only format that has this issue, except for "FMEObjects Gateway", which is not really a format. I don't understand why they didn't just follow the existing pattern and make it ESRIShapefile. :confused: